.github/workflows/deploy.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: '12.x'
-    - run: yarn install
+    - run: npm install
     - name: Deploy the extension
       run: |
-        yarn run grunt publish:chrome
-        yarn run grunt publish:firefox
+        npm run grunt publish:chrome
```

Comment:
You should use `npx` to run local packages. `npm run` is used to run scripts from `scripts` section in `package.json`.

Reply:
WIll update - forgot about this :/